docs/database-schema.md

```diff
@@ -0,0 +1,13 @@
+# Database Schema
+
+### wp_notifications table
+* id (int) 11 auto_increment `ID of the notification.`
+* sender `Sender of the notification.`
```

Comment:
The sender is becoming a model of its own. See #41 

So it may be a `sender_key` as well.

Reply:
Left a comment on #41 to understand what the sender data will look like so that I can update this as needed.